Initial Release

crestdatasystems opened the pull request:
### Features
New App: Google Threat Intelligence

### Manual Documentation

<details><summary>
Have you made any changes that should be documented in manual_readme_content.md?
</summary><br />

The following changes require documentation in `manual_readme_content.md`:

- New, updated, or removed [REST handlers](https://docs.splunk.com/Documentation/SOAR/current/DevelopApps/RESTHandlers)
- New, updated, or removed authentication methods, especially complex methods like OAuth
- Compatibility considerations with respect to deployment types (e.g. actions that cannot be run on cloud or an automation broker)
</details>

- [ ] I have verified that manual documentation has been updated where appropriate

### Other information
<!-- Delete this entire section if unused -->
<!-- Please add any other pertinent information you would like reviewers to know about your change -->

---
Please refer to our [Contribution Guide](https://github.com/splunk-soar-connectors/.github/blob/main/.github/CONTRIBUTING.md) for any questions on submitting a pull request.

Thanks for contributing!

crestdatasystems committed:
feat: initial release

crestdatasystems committed:
Updated manual readme file

crestdatasystems committed:
Fixed pre-commit

crestdatasystems commented:
Hello Splunk Team,
We have resolved the pre-commit failures on our end. It seems that some updates might be required from the Splunk side to address the remaining failing scenarios.

Please let us know if there’s anything still pending from our side. Also, do you have an estimated timeline for when the merge request will be merged?

mmomin-crest commented:
Hello @crestdatasystems, thank you for your contribution. We’ll add this to our queue and begin reviewing the PR shortly.

stutib-crest committed:
json changes

stutib-crest commented:
@crestdatasystems The description of the connector needs to be short, one of the possible descriptions could this:
"Supercharge Splunk SOAR with Google Threat Intelligence by integrating real-time IOCs, breach insights, and threat actor data from VirusTotal, Mandiant, and Google for automated, context-rich responses". Could you please update it? The rest of the information can be included in the manual_content_readme file of the connector.
Thanks

crestdatasystems committed:
Updated app description

stutib-crest committed:
updated pre-commit yaml

crestdatasystems commented:
@stutib-crest Thanks for the suggestion. We have updated the app description as per your recommendation and moved the previous description to manual_content_readme.

crestdatasystems committed:
Updated license information
